src/MagnumPlugins/AssimpImporter/AssimpImporter.cpp

```diff
@@ -1198,12 +1261,16 @@ Containers::Optional<MeshData> AssimpImporter::doMesh(const UnsignedInt id, Unsi
 }
 
 MeshAttribute AssimpImporter::doMeshAttributeForName(const Containers::StringView name) {
-    return _f ? _f->meshAttributesForName[name] : MeshAttribute{};
+    if(_f) {
```

Comment:
Since the mesh attribute names are not stored inside `_f` anymore, this branch wouldn't need to be here.

And I think it would be enough to have just `JointsAttribute` and `WeightAttribute` defined, and do the rest here with a plain old `if` cascade instead of iterating over arrays:

```cpp
    if(name == "JOINTS"_s) return JointsAttribute;
    if(name == "WEIGHTS"_s) return WeightsAttribute;
    return {};  
```